src/main/java/sx/blah/discord/handle/obj/Verification.java

```diff
@@ -0,0 +1,41 @@
+package sx.blah.discord.handle.obj;
+
+/**
+ * Represents a verification level for a guild
+ */
+public enum Verification {
```

Comment:
I think this would make more sense to be `VerificationLevel`. All references to it in guild should say level as well.